src/MOI_wrapper.jl

```diff
@@ -47,14 +47,57 @@ function MOI.add_constrained_variable(o::Optimizer, set::S) where {S <: MOI.Inte
     return (MOI.VariableIndex(col_idx), MOI.ConstraintIndex{MOI.SingleVariable, MOI.Interval{Float64}}(col_idx))
 end
 
-MOI.supports_constraint(::Optimizer, ::MOI.SingleVariable, ::MOI.Interval) = true
-
 function MOI.add_constraint(o::Optimizer, sg::MOI.SingleVariable, set::MOI.Interval)
     var_idx = Cint(sg.variable.value)
     _ = CWrapper.Highs_changeColBounds(o.model.inner, var_idx, Cdouble(set.lower), Cdouble(set.upper))
     return
 end
 
+MOI.supports_constraint(::Optimizer, ::MOI.SingleVariable, ::Union{MOI.Interval, MOI.LessThan, MOI.GreaterThan}) = true
+
+function MOI.get(o::Optimizer, ::MOI.ConstraintFunction, ci::MOI.ConstraintIndex{MOI.SingleVariable, <: Union{MOI.Interval, MOI.LessThan, MOI.GreaterThan}})
+    return MOI.SingleVariable(MOI.VariableIndex(ci.value))
+end
+
+function MOI.get(o::Optimizer, ::MOI.ConstraintSet, ci::MOI.ConstraintIndex{MOI.SingleVariable, MOI.Interval{Float64}})
+    var_idx = Cint(ci.value)
+    num_col = Ref{Cint}(0)
+    num_nz = Ref{Cint}(0)
+    lower = Ref{Cdouble}()
+    upper = Ref{Cdouble}()
+    _ = CWrapper.Highs_getColsByRange(o.model.inner, var_idx, var_idx, num_col, C_NULL, lower, upper, num_nz, C_NULL, C_NULL, C_NULL)
+    num_col[] == 1 || error("Unexpected HiGHS state, number of columns should be one for valid range, is $(num_col[])")
+    return MOI.Interval(lower[], upper[])
```

Comment:
It should be an error if the user queries a property of a constraint that wasn't added, e.g., if they added a `LessThan` and a `GreaterThan` constraint and there was never an `Interval` in the model added, this behavior isn't correct.

Reply:
what do you think is the best approach here, because there is always a value (even if it's +inf)